libs/beacon/include/beacon/beacon_setup_service.hpp

```diff
@@ -139,10 +148,11 @@ class BeaconSetupService
 
   /// Setup management
   /// @{
-
-  void QueueSetup(const SharedAeonExecutionUnit &beacon);
+  void StartNewCabinet(CabinetMemberList members, uint32_t threshold, uint64_t round_start,
```

Comment:
It is not switched away from a queue structure. Maybe QueueNewCabinet is a more appropriate name, but the change is mainly that the aeon_execution_unit it now created in beacon_setup_service itself and then pushed onto the queue rather than having it pre-constructed elsewhere and then pass to the beacon_setup_service to just push onto the queue.